src/CppUTest/SimpleString.cpp

```diff
@@ -478,6 +478,40 @@ SimpleString HexStringFrom(unsigned long value)
     return StringFromFormat("%lx", value);
 }
 
+#ifdef CPPUTEST_USE_LONG_LONG
+
+SimpleString StringFrom(cpputest_longlong value)
+{
+    return StringFromFormat("%lld", value);
+}
+
+SimpleString StringFrom(cpputest_ulonglong value)
+{
+    return StringFromFormat("%llu (0x%llx)", value, value);
+}
+
+SimpleString HexStringFrom(cpputest_longlong value)
+{
+    return StringFromFormat("%llx", value);
+}
+
+SimpleString HexStringFrom(cpputest_ulonglong value)
+{
+    return StringFromFormat("%llx", value);
+}
+
+SimpleString HexStringFrom(const void* value)
+{
+    return HexStringFrom((cpputest_ulonglong) value);
+}
+
+SimpleString HexStringFrom(void (*value)())
+{
+    return HexStringFrom((cpputest_ulonglong) value);
+}
+
+#else   /* CPPUTEST_USE_LONG_LONG */
+
 static long convertPointerToLongValue(const void* value)
```

Comment:
Not quite understand why this isn't needed when we add long long support. It feels unrelated ?